src/Driver/OCI8/Result.php

```diff
@@ -115,10 +120,18 @@ public function free(): void
      */
     private function fetch(int $mode)
     {
-        return oci_fetch_array(
-            $this->statement,
-            $mode | OCI_RETURN_NULLS | OCI_RETURN_LOBS
-        );
+        set_error_handler([$this, 'handleError'], E_WARNING);
```

Comment:
The coding style check doesn't register this as using the handleError method.  Is there a better way to call this to pass that coding style check?

Coding Standards (7.4)
Class Result contains unused private method handleError().